backend/src/v5/utils/responseCodes.js

```diff
@@ -94,6 +97,9 @@ ResponseCodes.templates = {
 	// Invalid Arguments
 	invalidArguments: { message: 'The arguments provided are not valid.', status: 400 },
 
+	// Resource not available
+	resourceNotAvailable: { message: 'Resource not available.', status: 410 },
```

Comment:
Can we rename this to `endpointDecomissioned`
```suggestion
	endpointDecomissioned: { message: 'Endpoint no longer available.', status: 410 },
```